docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -8,32 +8,63 @@ The Entity Analytics dashboard provides a centralized view of emerging insider t
 [sidebar]
 --
 
-* A https://www.elastic.co/pricing/[Platinum subscription] or higher is required.
-* To display host and user risk scores, you must <<turn-on-risk-engine, turn on the risk scoring engine>>.
+A https://www.elastic.co/pricing/[Platinum subscription] or higher is required.
 
 --
 
 The dashboard includes the following sections:
 
 * <<entity-kpis>>
-* <<entity-host-risk-scores>>
 * <<entity-user-risk-scores>>
-* <<entity-anomalies>> 
-
+* <<entity-host-risk-scores>>
+* <<entity-entities>>
+* <<entity-anomalies>>
 
+// screenshot to be updated
 [role="screenshot"]
 image::images/entity-dashboard.png[Entity dashboard]
 
 [[entity-kpis]]
 [float]
 == Entity KPIs (key performance indicators)
 
-Displays the total number of critical hosts, critical users, and anomalies. Select a link to jump to the Host risk table, User risk table, or Anomalies table. 
+Displays the total number of critical hosts, critical users, and anomalies. Select a link to jump to the **Hosts** page, **Users** page, or the **Anomalies** table. 
+
+[[entity-user-risk-scores]]
+[float]
+== User Risk Scores
```

Comment:
The diff shows the User Risk Scores section as added, but it's only been reordered (User Risk Scores and Host Risk Scores have swapped places in the UI, so I moved this section up), so no need to review it.